universal-application-tool-0.0.1/test/BrowserTest.java

```diff
@@ -31,24 +35,49 @@ protected TestBrowser provideBrowser(int port) {
   }
 
   @Test
-  public void test() {
+  public void homePage() {
     browser.goTo("http://localhost:" + play.api.test.Helpers.testServerPort());
-    assertTrue(browser.pageSource().contains("Your new application is ready."));
+    assertThat(browser.pageSource()).contains("Your new application is ready.");
+  }
+
+  @Test
+  public void applicantProgramList_selectApply_redirectsToEdit() {
+    Program program = insertProgram("My Program");
+    goTo(controllers.applicant.routes.ApplicantProgramsController.index(1L));
+    assertThat(browser.pageSource()).contains("My Program");
+
+    // Redirect when "apply" button is clicked.
+    String applyButtonId = "#apply" + program.id;
+    browser.$(applyButtonId).click();
+    assertThat("/" + browser.url())
```

Comment:
suggestion (non-blocking): this string cat is a little cryptic. Recommend moving into a named method, maybe in the superclass since it looks like something we'll want to do in a few tests

Reply:
I created a new `assertUrlEquals` method and will add it to the base class in #232